kubernetes/resource_kubernetes_service.go

```diff
@@ -159,6 +202,11 @@ func resourceKubernetesServiceRead(d *schema.ResourceData, meta interface{}) err
 		return err
 	}
 
+	err = d.Set("load_balancer_ingress", flattenLoadBalancerIngress(svc.Status.LoadBalancer.Ingress))
```

Comment:
The K8S always returned empty struct consistently - i.e. `svc.Status` was never `nil` so I don't think so.